app/Http/Controllers/Api/v1/User/Controllers/GenerateTokenController.php

```diff
@@ -42,13 +44,42 @@ public function __invoke(Request $request): JsonResponse
         try {
             $user = $this->userLookup->getUpdatedUser($user, $azureUser->principal_name);
         } catch (ClientExceptionInterface) {
+            $log = new Log();
+            $log->user_id = $user->id;
+            $log->severity = '5';
+            $log->category = 'mensa';
```

Comment:
Just like severity, I'd like to see this in an enum or something. manually typing `mensa` every time seems error prone.

Reply:
the categories are currently an enum (migration wise). Will check this too

Reply:
```php
class Category {
    ...
    public static $MENSA = 'mensa';
}

$log->category = Category::MENSA;
```